proto/Makefile

```diff
@@ -26,16 +26,26 @@ ifeq (,$(strip $(PROTOC_VER)))
 PROTOC_VER := 27.2
 endif
 
+# Only set PROTOC_OS if it has an empty value.
+ifeq (,$(strip $(PROTOC_OS)))
```

Comment:
This should be done with `PROTOC_OS ?= $(shell uname -s)`.

This also goes for other fields.